aptos-move/framework/table-natives/src/lib.rs

```diff
@@ -287,7 +287,17 @@ fn charge_load_cost(
 
     match loaded {
         Some(Some(num_bytes)) => {
-            context.charge(COMMON_LOAD_BASE_NEW + COMMON_LOAD_PER_BYTE * num_bytes)
+            if context.gas_feature_version() >= 12 {
```

Comment:
stupid question, why does this not go through IoPricing utilities, like everything else?

Reply:
I was just keeping the structure.. 
Indeed it'll be nice if we can remove this shit as well: https://github.com/aptos-labs/aptos-core/blob/b1f3abec3988b0c791fe19b5c9663b75ec1d9393/aptos-move/aptos-vm/src/gas.rs#L63

Reply:
can you at least add a TODO?

seems like it is very easy to miss this place when modifying gas schedule.